pkg/api/dashboard_test.go

```diff
@@ -26,6 +26,10 @@ func TestGetHomeDashboard(t *testing.T) {
 		HomePath: "../../",
 	})
 
+	hs := &HTTPServer{
+		Cfg: cfg,
+	}
+
```

Comment:
Can you move this until after the error check? It's confusing now that the error check is located away from line 25.